### PR TITLE
[FIX] account_payment_pro: FIX in compute amount_company_currency when already having a payment posted

### PR DESCRIPTION
**Pull request by rov-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/530).

File: account_payment_pro/models/account_payment.py [hunk 1]
```diff
@@ -259,9 +258,11 @@ def _compute_amount_company_currency(self):
...
                 amount_company_currency = rec.force_amount_company_currency
             else:
-                amount_company_currency = rec.currency_id._convert(
-                    rec.amount, rec.company_id.currency_id,
-                    rec.company_id, rec.date)
+                amount_company_currency = rec.amount_company_currency_signed
+                if not rec.amount_company_currency_signed:
+                    amount_company_currency = rec.currency_id._convert(
+                        rec.amount, rec.company_id.currency_id,
+                        rec.company_id, rec.date)
```
Comment (cav-adhoc) [inline account_payment_pro/models/account_payment.py]: @rov-adhoc creo que así queda con menos líneas y no se asigna 2 veces, por si suma
amount_company_currency = rec.amount_company_currency_signed or rec.currency_id._convert(
    rec.amount, rec.company_id.currency_id, rec.company_id, rec.date
)

**Comment by cav-adhoc** [comment]
@roboadhoc r+

**Comment by cav-adhoc** [comment]
@roboadhoc r+

**Pull request closed without merging**